xarray/tests/test_missing.py

```diff
@@ -451,10 +451,6 @@ def test_ffill_bfill_dask(method):
     expected = numpy_method("time", limit=3)
     assert_equal(actual, expected)
 
-    # limit < axis size
-    with pytest.raises(NotImplementedError):
```

Comment:
if this works  now lets compare the result of `dask_method` with `numpy_method` like the other tests.